Druid/DRCogitSendMarshallHoisting.class.st

```diff
@@ -21,8 +21,8 @@ DRCogitSendMarshallHoisting >> applyTo: cfg [
 	toMerge := cfg instructions select: [ :e | e isPop or: [ e isCogitSendMarshall ] ].
 	toMerge size <= 1 ifTrue: [ ^ self "nothing to do" ].
 
-	cfg buildDominatorTree.
-	dominator := cfg tree dominatorOfAll: toMerge.
+	cfg invalidateDominatorTree.
```

Comment:
Niceeee 🤩 

We should call automatically this method on any change of predecessors / successors on any block.

Then, we don't need to force it any time that the dominator tree _could_ be used.

Reply:
https://github.com/Alamvic/druid/issues/140 :octocat:

Reply:
Yes, thanks for the issue :)